src/core/EventDispatcher.js

```diff
@@ -76,6 +76,13 @@ THREE.EventDispatcher.prototype = {
 
 	dispatchEvent: function ( event ) {
 
+		if (!event.target) {
```

Comment:
For code readability, would be better to be more explicit here.... `event.target !== undefined`.